autoware.repos

```diff
@@ -71,3 +67,39 @@ repositories:
     type: git
     url: https://github.com/tier4/image_pipeline.git
     version: 47964112293eb19f9f57254b2e6b68706954cc63
+  external/muSSP:
+      type: git
+      url: https://github.com/tier4/muSSP.git
+      version: tier4/main
+  external/ndt_omp:
+      type: git
+      url: https://github.com/tier4/ndt_omp.git
+      version: tier4/main
+  external/eagleye:
+      type: git
+      url: https://github.com/MapIV/eagleye.git
+      version: autoware-main
+  external/morai_msgs:
+      type: git
+      url: https://github.com/MORAI-Autonomous/MORAI-ROS2_morai_msgs.git
+      version: main
+  external/autoware_launch:
+      type: git
+      url: https://github.com/autowarefoundation/autoware_launch.git
+      version: main
+  external/transport_drivers:
```

Comment:
This repository already exists in this repos file - duplicates causes issues so please remove this one.